### PR TITLE
Add String.prototype methods to the chain object

### LINKED ISSUE #371: Add String.prototype methods to the chain object
`toLowerCase` etc. for convenience. Already started working on this.

### PR DESCRIPTION
**Pull request by epeli**
Closes #371

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,4 +107,30 @@ function fn2method(key, fn) {
 // Copy functions to instance methods for chaining
 for (var key in s) fn2method(key, s[key]);
 
+fn2method("tap", function tap(string, fn) {
+  return fn(string);
+});
+
+function prototype2method(methodName) {
+  fn2method(methodName, function() {
```
Comment (stoeffel) [inline index.js]:
```js
fn2method(methodName, function(context) {
    var args = Array.prototype.slice.call(arguments, 1);
```

File: index.js [hunk 2]
```diff
@@ -110,4 +107,30 @@ function fn2method(key, fn) {
...
+
+var prototypeMethods = [
+  "toUpperCase",
+  "toLowerCase",
+  "split",
+  "replace",
+  "slice",
+  "substring",
+  "substr",
+  "concat",
```
Comment (stoeffel) [inline index.js]: trailing `,`

**Comment by stoeffel** [comment]
Two small code style things. But the rest looks good.

File: index.js (hunk 2 above)
Comment (epeli) [inline index.js]: fixed

File: index.js (hunk 1 above)
Comment (epeli) [inline index.js]: check

**Pull request merged**